### ISSUE TITLE
BUG - updateById can not update json columns in mysql

### ISSUE THREAD

**Issue by Diluka** [typeorm/typeorm#1105]
updateById can not update json columns in mysql

```ts
data={
    contents:[
        {type:1,content:"test"}
    ]
}

repo.updateById(id,data)
```

generated sql shows no any column is updated
```sql
UPDATE `table` SET  WHERE `id`=?  # You have an error in your SQL syntax
```

**Comment by Diluka** [comment]
@pleerock https://github.com/Diluka/typeorm-issue-1105
the error only reproduce in production version using `0.1.1`

**Comment by pleerock** [comment]
its a known issue and fix will be released in 0.1.2. duplicate of #1042
